<launch>
	<node pkg="gitagent" type="environment.py" name="env_node" output="log" required="true" launch-prefix="xterm -e">
		<param name="maxFires" value="40" />
		<param name="maxIntensity" value="50" />
		<param name="maxVictim" value="10" />
	</node>
</launch>
